pkg/antctl/raw/multicluster/common/common.go

```diff
@@ -0,0 +1,415 @@
+			return err
+		}
+		fmt.Fprintf(cmd.OutOrStdout(), "ClusterClaim \"%s\" deleted in Namespace %s\n", name, namespace)
+	}
+	return nil
+}
+
+func deleteClusterSet(cmd *cobra.Command, k8sClient client.Client, namespace string, clusterSet string) error {
+	if err := k8sClient.Delete(context.TODO(), newClusterSet(clusterSet, namespace, "", "", "", "", "")); err != nil && !apierrors.IsNotFound(err) {
+		fmt.Fprintf(cmd.OutOrStdout(), "Failed to delete ClusterSet \"%s\": %v\n", clusterSet, err)
+		return err
+	}
+	fmt.Fprintf(cmd.OutOrStdout(), "ClusterSet \"%s\" deleted in Namespace %s\n", clusterSet, namespace)
+	return nil
+}
+
+func deleteSecret(cmd *cobra.Command, k8sClient client.Client, namespace string) error {
+	secretList := &corev1.SecretList{}
+	if err := k8sClient.List(context.TODO(), secretList, client.InNamespace(namespace)); err != nil {
+		fmt.Fprintf(cmd.OutOrStdout(), "Failed to list Secrets in Namespace %s: %v\n", namespace, err)
+	}
+
+	for _, secret := range secretList.Items {
+		if secret.Annotations[CreateByAntctlAnnotation] != "true" {
+			continue
+		}
+
+		if err := k8sClient.Delete(context.TODO(), &secret); err != nil {
+			fmt.Fprintf(cmd.OutOrStdout(), "Failed to delete Secret \"%s\": %v\n", secret.Name, err)
+			return err
+		}
+		fmt.Fprintf(cmd.OutOrStdout(), "Secret \"%s\" deleted in Namespace %s\n", secret.Name, namespace)
+	}
+
+	return nil
+}
+
+func deleteRoleBinding(cmd *cobra.Command, k8sClient client.Client, namespace string) error {
+	roleBindingList := &rbacv1.RoleBindingList{}
+	if err := k8sClient.List(context.TODO(), roleBindingList, client.InNamespace(namespace)); err != nil {
+		fmt.Fprintf(cmd.OutOrStdout(), "Failed to list RoleBindings in Namespace %s: %v\n", namespace, err)
+	}
+
+	for _, roleBinding := range roleBindingList.Items {
+		if roleBinding.Annotations[CreateByAntctlAnnotation] != "true" {
+			continue
+		}
+
+		if err := k8sClient.Delete(context.TODO(), &roleBinding); err != nil {
+			fmt.Fprintf(cmd.OutOrStdout(), "Failed to delete RoleBinding \"%s\": %v\n", roleBinding.Name, err)
+			return err
+		}
+		fmt.Fprintf(cmd.OutOrStdout(), "RoleBinding \"%s\" deleted in Namespace %s\n", roleBinding.Name, namespace)
+	}
+
+	return nil
+}
+
+func deleteServiceAccount(cmd *cobra.Command, k8sClient client.Client, namespace string) error {
+	serviceAccountList := &corev1.ServiceAccountList{}
+	if err := k8sClient.List(context.TODO(), serviceAccountList, client.InNamespace(namespace)); err != nil {
+		fmt.Fprintf(cmd.OutOrStdout(), "Failed to list ServiceAccounts in Namespace %s: %v\n", namespace, err)
+	}
+
+	for _, serviceAccount := range serviceAccountList.Items {
+		if serviceAccount.Annotations[CreateByAntctlAnnotation] != "true" {
+			continue
+		}
+
+		if err := k8sClient.Delete(context.TODO(), &serviceAccount); err != nil {
+			fmt.Fprintf(cmd.OutOrStdout(), "Failed to delete ServiceAccount \"%s\": %v\n", serviceAccount.Name, err)
+			return err
```

Comment:
Should we ignore not found error for all deletion actions? this not found case might be rare, but I feel it's still possible to happen.

Reply:
If we choose not to return at a single error but always try to delete all resources, I feel not important to ignore "not-found".